libs/datasets/timeseries.py

```diff
@@ -639,9 +621,9 @@ def get_counties(
         # metrics which no longer have real values are excluded.
         latest_df, provenance = self._get_latest_and_provenance_for_locations(location_ids)
 
-        return MultiRegionTimeseriesDataset.from_combined_dataframe(
-            pd.concat([ts_df, latest_df], ignore_index=True), provenance=provenance
-        )
+        return MultiRegionTimeseriesDataset.from_timeseries_df(
+            ts_df, provenance=provenance
+        ).append_latest_df(latest_df)
```

Comment:
Small but would love to find a way to remove this eventually.  Don't think we're quite there yet - just a thought!